WMRC Blog Post (#2861)
agrc/gis.utah.gov: src/content/blog/2024-09-30-dwmrc-and-ugrc-release-map-to-help-utahns-recycle-better.mdx

---

> title: Dwmrc and ugrc release map to help utahns recycle better

Reviewer: ```suggestion
title: DWMRC and UGRC release map to help Utahns recycle better
```

Once the tabular data are organized and cleaned, we use pandas scripts to perform calculations and gather statistics from the reports. These scripts run fully automated so that the statistics will be updated as new data are entered into the database. The fully automated process saves time, money, and helps to prevent human error.

Even if you don't have a haunted vase from a long-lost relative lying around (perhaps we should all check?), there are plenty of reasons why having access to this data is a good thing. First and foremost, the map serves as a reference for where you can dispose of any other haunted articles you have in your possession and regular items such as paint cans, batteries, and furniture. The map also points out recycling centers and has been used by organizations such as the Utah Recycling Alliance to help spread awareness about recycling efforts ([2](https://www.utahrecyclingalliance.org/post/utah-s-new-recycling-data-initiative)). Just knowing where and how to recycle can help people to actually do it. 

> Another benefit of this project is transparency. Utahns deserve to know how the government is using taxpayer's dollars to promote effective waste management practices in our state. With these data products, the average person can see important metrics such as the percentage of waste materials that were recycled in Utah and how many gallons of used oil were collected for recycling. Having these metrics publicly available adds an extra layer of accountability for waste management facilities, DWMRC.

Reviewer 2: ```suggestion
Another benefit of this project is transparency. Utahns deserve to know how the government is using their tax dollars to promote effective waste management practices in our state. With these data products, the average person can see important metrics such as the percentage of waste materials that were recycled in Utah and how many gallons of used oil were collected for recycling. Having these metrics publicly available adds an extra layer of accountability for waste management facilities, DWMRC.
```

Also, the last sentence here feels like there was something more that was meant to go on there.

> This project is one of many collaborative efforts between UGRC and other state and federal agencies. At UGRC, our mission is to “encourage and facilitate the effective use of geospatial information and technology for Utah.” ([3](https://gis.utah.gov/about/) Working with DWMRC to produce these data products is just one of the many ways we advocate for data-driven decision making and public access to geospatial data in Utah. 

Reviewer: internal links should be relative [example](https://github.com/agrc/gis.utah.gov/blob/main/README.md#link-to-page)
```suggestion
This project is one of many collaborative efforts between UGRC and other state and federal agencies. At UGRC, our mission is to “encourage and facilitate the effective use of geospatial information and technology for Utah.” ([3](/about/) Working with DWMRC to produce these data products is just one of the many ways we advocate for data-driven decision making and public access to geospatial data in Utah. 
```

> This project is one of many collaborative efforts between UGRC and other state and federal agencies. At UGRC, our mission is to “encourage and facilitate the effective use of geospatial information and technology for Utah.” ([3](/about/) Working with DWMRC to produce these data products is just one of the many ways we advocate for data-driven decision making and public access to geospatial data in Utah. 

Reviewer 2: ```suggestion
This project is one of many collaborative efforts between UGRC and other state agencies. At UGRC, our mission is to “encourage and facilitate the effective use of geospatial information and technology for Utah.” ([3](/about/) Working with DWMRC to produce these data products is just one of the many ways we advocate for data-driven decision making and public access to geospatial data in Utah. 
```
We generally don't do projects with federal agencies like we do for state agencies. We partner with them for specific things (lidar, PLSS) but we're not creating solutions for federal agencies.

> You can learn more about our other projects on UGRC's [blog](https://gis.utah.gov/blog/) and in our [quarterly newsletter](https://cdn.forms-content.sg-form.com/61e82d28-f0d5-11ec-847c-a26809937a38). For more information on waste management in Utah and these data products in particular, please reach out to Stevie Norcross at DWMRC (stevienorcross@utah.gov) or Jake Adams at UGRC (jdadams@utah.gov).

Reviewer: 1. internal links should be relative [example](https://github.com/agrc/gis.utah.gov/blob/main/README.md#link-to-page)
2. Please use the short link for the news letter
3. emails should be linked from contacts and not placed directly in content [example](https://github.com/agrc/gis.utah.gov/blob/main/README.md#contact-information)

```suggestion
You can learn more about our other projects on UGRC's [blog](/blog/) and in our [quarterly newsletter](https://s.utah.gov/ugrc-newsletter). For more information on waste management in Utah and these data products in particular, please reach out to Stevie Norcross at DWMRC (stevienorcross@utah.gov) or Jake Adams at UGRC (jdadams@utah.gov).
```

> You can learn more about our other projects on UGRC's [blog](https://gis.utah.gov/blog/) and in our [quarterly newsletter](https://cdn.forms-content.sg-form.com/61e82d28-f0d5-11ec-847c-a26809937a38). For more information on waste management in Utah and these data products in particular, please reach out to Stevie Norcross at DWMRC (stevienorcross@utah.gov) or Jake Adams at UGRC (jdadams@utah.gov).

Reviewer: The comment that had this was resolved. adding a new one :)

1. emails should be linked from contacts and not placed directly in content [example](https://github.com/agrc/gis.utah.gov/blob/main/README.md#contact-information)
